.github/workflows/js_client_release.yaml

```
@@ -17,7 +21,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version-file: .nvmrc
+          node-version-file: .js-client/nvmrc
```

Is that correct? Shouldn't `it be smth like this:
```suggestion
          node-version-file: js-client/.nvmrc
```

Also, wouldn't changing `working-directory` already fix the issue?

---

ahh yes
as for `working-directory`, from what I've seen it works only with `runs` and may not work with `uses`